api/src/main/java/org/eclipse/microprofile/lra/annotation/LRA.java

```diff
@@ -40,6 +40,12 @@
  * performs will carry the same header so that the receiving resource knows that it
  * is inside an LRA context (typically achieved using JAX-RS client filters).
  *
+ * Note that if an LRA is propagated to a resource that is not annotated with any
+ * particular LRA behaviour then the LRA will be suspended. However, if the method
+ * performs any outgoing JAX_RS requests then the LRA will be propagated to
```

Comment:
As per the call today you should make the wording in the Javadoc more explicit:
service A can start an LRA by using LRAClient or an annotation, 
then that it can call a (potentially) non-LRA aware service B (i.e. one that is not marked up with an @LRA annotation) and 
that service B can call an LRA aware service C (i.e. one that uses LRAClient to join the LRA or has the @LRA annotation) and that in service C the LRA should be available